config/application.rb

```diff
@@ -87,5 +81,29 @@ def self.fog_credentials
         fog_public: true
       }
     end
+
+    config.action_mailer.default_url_options = {
+      host:     ENV['CANONICAL_HOST'],
+      port:     ENV['CANONICAL_PORT'],
+      protocol: ENV['FORCE_SSL'] ? 'https' : 'http'
+    }
+
+    config.force_ssl = ENV.has_key?('FORCE_SSL')
```

Comment:
@gdpelican doing force_ssl for all environments here. but no it does not default to true :)